src/app/ngrx/reducers/episode-ranks.reducer.spec.ts

```diff
@@ -6,7 +6,7 @@ import {
   ep0AgentNameRanks,
   ep0AgentNameDownloads
 } from '../../../testing/downloads.fixtures';
-import { GroupType, GROUPTYPE_AGENTNAME, METRICSTYPE_TRAFFICSOURCES, episodeRanksKey } from './models';
+import { GroupType, GROUPTYPE_AGENTNAME, METRICSTYPE_TRAFFICSOURCES, episodeRanksId } from './models';
```

Comment:
This is a naming change. I just started using the id field instead of adding a separate combo-key and pointing to it with the `selectId` helper.